Lib/test/_test_multiprocessing.py

```diff
@@ -891,6 +891,60 @@ def test_forkserver_sigkill(self):
         if os.name != 'nt':
             self.check_forkserver_death(signal.SIGKILL)
 
+    @staticmethod
+    def _exit_process():
+        sys.exit(0)
+
+    def test_forkserver_auth_is_enabled(self):
+        if self.TYPE == "threads":
+            self.skipTest(f"test not appropriate for {self.TYPE}")
+        if multiprocessing.get_start_method() != "forkserver":
+            self.skipTest("forkserver start method specific")
+
+        forkserver = multiprocessing.forkserver._forkserver
+        forkserver.ensure_running()
+        self.assertTrue(forkserver._forkserver_pid)
+        authkey = forkserver._forkserver_authkey
+        self.assertTrue(authkey)
+        self.assertGreater(len(authkey), 15)
+        addr = forkserver._forkserver_address
+        self.assertTrue(addr)
+
+        # First, demonstrate that a raw auth handshake as Client makes
+        # does not raise an error.
```

Comment:
I *think* you meant the following.

```suggestion
        # First, demonstrate that a raw auth handshake as Client
        # does not raise an error.
```

Reply:
reworded.